packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -461,3 +483,67 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     );
   }
 }
+
+/// Returns editing intent for selector from NSStandardKeyBindingResponding
+/// if the selector is recognized.
+Intent? intentForMacOSSelector(String selectorName) {
+  const Map<String, Intent> selectorToIntent = <String, Intent>{
```

Comment:
Nit: Should this map be declared outside of the method?  Maybe it doesn't matter.

Reply:
I had it declared outside first and then was suggested to put it in the method (or I misunderstood the suggestion). I don't have strong feelings about this. If you do I can put it outside.

Reply:
I don't, I'm happy with it inside :+1: